app/src/main/java/com/mho/training/sources/MovieDataSource.kt

```diff
@@ -19,92 +22,113 @@ class MovieDataSource(
 
     //region Override Methods & Callbacks
 
-    override suspend fun getTopRatedMovieList(region: String): Result<List<Movie>> = withContext(Dispatchers.IO) {
+    override suspend fun getTopRatedMovieList(region: String): DataResult<List<Movie>> = withContext(Dispatchers.IO) {
         safeApiCall(
             call = { requestTopRatedMovieList(region) },
             errorMessage = resources.getString(R.string.error_unable_to_fetch_movies)
```

Comment:
Same as https://github.com/proyectox123/architect_coders/pull/8/files#r367758860

Reply:
For now, for movies I prefer use resources for a specific implementation.